src/main/java/processor/Parser.java

```diff
@@ -0,0 +1,58 @@
+package processor;
+
+import commands.CommandType;
+import commands.Commander;
+import commands.Command;
+
+/**
+ * Handles the parsing of user input.
+ */
+public class Parser {
```

Comment:
Is it possible to also handle the parsing of arguments for each command in the Parser class too, so that the commands themselves do not have to handle any parsing, and take in necessary information via constructor/method parameters instead?